abrams_strogatz/lang_comp_bilinguals.py

```diff
@@ -0,0 +1,300 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Initial condition. Randomly distributed speakers.
+
+
+def initial_cond(m: int, n: int, pa: float = 1./3., pb: float = 1./3.) -> np.array:
+    """Creates the initial (mxn) array.
```

Comment:
10:   1 D401 First line should be in imperative mood [flake8-docstrings]
```suggestion
    """Create the initial (mxn) array representing a population of speakers.
```

Reply:
I have applied the requested changes and modified some docstrings.